# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/cdrtools/linux-2.6.7-__user.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libscg/scsi-linux-sg.c.orig	2002-10-21 19:20:35.000000000 +0300
+++ ./libscg/scsi-linux-sg.c	2004-06-27 15:23:02.670972688 +0300
@@ -78,6 +78,10 @@
 #include "scsi/scsi.h"
 #endif
 
+#if LINUX_VERSION_CODE >= 0x020607
+#include <linux/compiler.h>
+#endif
+
 #include "scsi/sg.h"
 
 #undef sense			/* conflict in struct cdrom_generic_command */
